podman_compose.py

```diff
@@ -937,29 +935,16 @@ def get_net_args_from_networks(compose, cnt):
     service_name = cnt["service_name"]
 
     aliases_on_container = [service_name]
-    # NOTE: from podman manpage:
-    # NOTE: A container will only have access to aliases on the first network
-    #       that it joins. This is a limitation that will be removed in a later
-    #       release.
     aliases_on_container.extend(cnt.get("_aliases", []))
-    aliases_on_net = []
-
-    # TODO: add support for per-interface aliases
-    #  See https://docs.docker.com/compose/compose-file/compose-file-v3/#aliases
-    #  Even though podman accepts network-specific aliases (e.g., --network=bridge:alias=foo,
-    #  podman currently ignores this if a per-container network-alias is set; as pdoman-compose
-    #  always sets a network-alias to the container name, is currently doesn't make sense to
-    #  implement this.
+
     multiple_nets = cnt.get("networks", {})
     if not multiple_nets:
         if not compose.default_net:
             # The bridge mode in podman is using the `podman` network.
             # It seems weird, but we should keep this behavior to avoid
             # breaking changes.
-            net_args.append("--network=bridge")
-            if mac_address:
-                net_args.append(f"--mac-address={mac_address}")
```

Comment:
I think we lose mac-address here. Is this deliberate? If yes, please split to separate commit with explanation.

Reply:
I just missing the mac_address here.

The top-level mac_address should be part of networks, and docker >= 25.0 will ignore this attribute
in favor of the network-scoped mac_address. see [compose spec](https://github.com/compose-spec/compose-spec/blob/main/05-services.md#mac_address-1) for more detail.
    
I'll set the mac_address in the bridge mode here for backward compatibility.